doc/reference/trd1-trds.md

```diff
@@ -247,23 +249,24 @@ from TinyOS Enhancement Proposal 1 (TEP 1).
 7. Author's Address
 ====================================================================
 
-```
-Philip Levis
-409 Gates Hall
-Stanford University 
-Stanford, CA 94305
+    Philip Levis
+    409 Gates Hall
+    Stanford University
+    Stanford, CA 94305
 
-phone - +1 650 725 9046
+    phone - +1 650 725 9046
 
-email - pal@cs.stanford.edu
-```
+    email - pal@cs.stanford.edu
 
 
 8. Citations
 ====================================================================
 
-[TRD1]: <trd1-trds.md>
-[Markdown]: <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links>
+<a name="TRD1" />[TRD1]
```

Comment:
How about just not having a Citations section and use Markdown's built-in citations as @phil-levis originally did (except he accidentally used `[TRD]` instead of `[TRD1]`. GitHub will render these as direct links from the text, which is very WWW